### PR TITLE
Implement BuildLogs

### PR DESCRIPTION
**Pull request by vgough**
Adds Logs to Build result, and implements BuildLogs function.

Based on current API described at http://readme.drone.io/api/build-endpoint/

### CODE REVIEW

**Comment by bradrydzewski** [comment]
note that the readme.drone.io is legacy and the latest documentation is available at http://docs.drone.io/api-overview/

File: drone/types.go [hunk 1]
```diff
@@ -98,6 +99,18 @@ type (
 		Children []*Proc           `json:"children,omitempty"`
 	}
 
+	// Job represents a build job.
+	Job struct {
+		ID          int               `json:"id"`
```
Comment (bradrydzewski) [inline drone/types.go]: the jobs structure no longer exists. It was removed in Drone version 0.6

File: drone/types.go [hunk 2]
```diff
@@ -78,6 +78,7 @@ type (
 		Reviewer  string  `json:"reviewed_by"`
 		Reviewed  int64   `json:"reviewed_at"`
 		Procs     []*Proc `json:"procs,omitempty"`
+		Jobs      []*Job  `json:"jobs,omitempty"`
```
Comment (bradrydzewski) [inline drone/types.go]: jobs attribute no longer exists as of version 0.6

**Comment by vgough** [comment]
Ok, I'll use a private branch for now, since I still need access to logs.

**Pull request closed without merging**